src/ServiceControl.TransportAdapter/NServiceBusMetricReport.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Metrics;
+using Metrics.Json;
+using Metrics.MetricData;
+using Metrics.Reporters;
+using NServiceBus;
+using NServiceBus.Extensibility;
+using NServiceBus.Logging;
+using NServiceBus.Routing;
+using NServiceBus.Support;
+using NServiceBus.Transport;
+
+namespace ServiceControl.TransportAdapter
+{
+    using NServiceBus.Raw;
+
+    class NServiceBusMetricReport<TServiceControl> : MetricsReport
+        where TServiceControl : TransportDefinition, new()
+    {
+        public NServiceBusMetricReport(string name, Guid hostId, string serviceControlMonitoringQueue,
+            Action<TransportExtensions<TServiceControl>> backendTransportCustomization)
+        {
+            senderConfig = RawEndpointConfiguration.CreateSendOnly($"{name}.MetricsSender");
+            var transport = senderConfig.UseTransport<TServiceControl>();
+            backendTransportCustomization(transport);
+
+            destination = new UnicastAddressTag(serviceControlMonitoringQueue);
+
+            headers["NServiceBus.Monitoring.NodeType"] = "ServiceControl.TransportAdapter";
+            headers[Headers.OriginatingMachine] = RuntimeEnvironment.MachineName;
+            headers[Headers.OriginatingHostId] = hostId.ToString("N");
+            headers[Headers.EnclosedMessageTypes] = "NServiceBus.Metrics.MetricReport"; // without assembly name to allow ducktyping
+            headers[Headers.ContentType] = ContentTypes.Json;
+        }
+
+        public async Task Start()
+        {
+            dispatcher = await RawEndpoint.Start(senderConfig).ConfigureAwait(false);
+        }
+
+        public Task Stop()
+        {
+            var sender = dispatcher;
+            dispatcher = null;
+            return sender.Stop();
+        }
+
+        public void RunReport(MetricsData metricsData, Func<HealthStatus> healthStatus, CancellationToken token)
+        {
+            if (dispatcher == null)
+            {
+                return;
+            }
+            RunReportAsync(metricsData)
```

Comment:
Nah, too much work. The default interval is 30 seconds. We should be able to complete the send in 30 seconds I guess. Plus the receiving code can't make any assumptions on the message order anyway.